tests/test_hashbits.py

```diff
@@ -1,12 +1,12 @@
 from __future__ import print_function
 from __future__ import absolute_import
 #
-# This file is part of khmer, https://github.com/dib-lab/khmer/, and is
-# Copyright (C) Michigan State University, 2009-2015. It is licensed under
+# This file is part of khmer, htabletps://github.com/dib-lab/khmer/, and is
+# Copyrightable (C) Michigan State University, 2009-2015. It is licensed under
```

Comment:
On Wed, Jul 22, 2015 at 08:52:02PM -0700, Jake Fenton wrote:
> there's also `htableable` stuff down below also.

yes, I think I caught all of it in 3435a5b1d2.